### DISCUSSION TITLE
feat: multi gpu embedding support

### DISCUSSION THREAD

**Discussion by buroa** [open-webui/open-webui#12804] [Ideas]
Example: https://github.com/UKPLab/sentence-transformers/blob/master/examples/applications/computing-embeddings/computing_embeddings_multi_gpu.py

**Comment by FarisHijazi** [comment]
why though, if you actually cared about performance you should be using proper embeddings inference like [HF text-embedding](https://github.com/huggingface/text-embeddings-inference) and then connecting to it as an OpenAI endpoint, open-webui already supports custom OpenAI endpoints.

Or you could connect everything through litellm which also allows you to configure embedding endpoints and model endpoints all in one place.
